### PR TITLE
Minor changes to readme and spark streaming sample

### PR DESCRIPTION
**Pull request by navalev**
- Update readme beta info on maven dependency
 - update gitignore
 - update spark streaming sample to set async to false by default as a temporary workaround to improve performance #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ version = 1.0.0-Beta-01
 ```
 
 **In Maven**:
+
+Note that the jar is in beta and not available yet in public maven. clone this repository and build it localy to add it to your local maven repository, or use the [pre-built jars](lib/)
```
Comment (michazag) [inline README.md]: -> Note: (add the '->' is for .md formatting)

. clone - . Clone (capitalize)

File: samples/src/main/scala/SparkStreamingKustoSink.scala [hunk 2]
```diff
@@ -47,7 +47,7 @@ object SparkStreamingKustoSink {
       .option(KustoOptions.KUSTO_TABLE, "Your Kusto Destination Table")
       .option(KustoOptions.KUSTO_AAD_CLIENT_ID, "Your Client ID")
       .option(KustoOptions.KUSTO_AAD_CLIENT_PASSWORD, "Your secret")
-      .option(KustoOptions.KUSTO_WRITE_ENABLE_ASYNC, "true")
+      .option(KustoOptions.KUSTO_WRITE_ENABLE_ASYNC, "false")
```
Comment (michazag) [inline samples/src/main/scala/SparkStreamingKustoSink.scala]: This is the default, so you can just omit this option.

File: README.md (hunk 1 above)
Comment (navalev) [reply README.md]: done.

File: samples/src/main/scala/SparkStreamingKustoSink.scala [hunk 3]
```diff
@@ -9,6 +9,10 @@ import org.apache.spark.sql.functions._
 /** ************************************************/
 /*          STREAMING SINK EXAMPLE                */
 /** ************************************************/
+
+// To enable faster ingestion into kusto, set a  minimal value for the batching ingestion policy:
```
Comment (ohadbitt) [inline samples/src/main/scala/SparkStreamingKustoSink.scala]: This should be followed with a warning for pricing and maybe even a link to the docs : https://kusto.azurewebsites.net/docs/concepts/batchingpolicy.html?q=batching

Comment (michazag) [reply samples/src/main/scala/SparkStreamingKustoSink.scala]: Would be better to point to the external documentation at https://docs.microsoft.com/en-us/azure/kusto/concepts/ 
The only problem is that the batching policy section was not published. Vladik made the necessary change already, I just don't see it reflected on the documentation site yet

**Pull request merged**